cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocConf.java

```diff
@@ -93,8 +93,8 @@ final class DataprocConf {
   public static final String COMPUTE_HTTP_REQUEST_READ_TIMEOUT = "compute.request.read.timeout.millis";
   private static final int COMPUTE_HTTP_REQUEST_READ_TIMEOUT_DEFAULT = 20000;
 
-  // If true, artifacts will not be cached in GCS regardless of cConf setting.
-  static final String DISABLE_GCS_CACHING = "disableGCSCaching";
+  // If false, artifacts will not be cached in GCS regardless of cConf setting.
+  static final String GCS_CACHE_ENABLED = DataprocUtils.GCS_CACHE_ENABLED;
```

Comment:
Why need to redefine it? Just use DataprocUtils.GCS_CACHE_ENABLED directly.

Reply:
ack